### PR TITLE
Modified YAML Test Cases - Dec 28th

### PR DESCRIPTION
**Pull request by ghost**
Modified following scripts as per latest testplan:
1. TC-BI-1.1
2. TC-BOOL-1.1
3. TC-CC-1.1
4. TC-CC-2.1
5. TC-CC-3.1
6. TC-CC-6.2
7. TC-CC-7.1
8. TC-CC-7.2
9. TC-CC-9.1
10. TC-CC-9.2
11. TC-CC-9.3
12. TC-DIAGTH-1.1
13. TC-EMR-1.1
14. TC-LVL-1.1
15. TC-ILL-1.1
16. TC-TM-1.1
17. TC-TSUIC-1.1
18. TC-TSTAT-1.1
19. TC-WNCV-1.1
20. TC-MC-1.1
21. TC-OCC-1.1
22. TC-OO-1.1
23. TC-PCC-1.1
24. TC-PRS-1.1
25. TC-RH-1.1
26. TC-FLW-1.1

Executed all YAML test scripts, attached logs for reference:
[Media_cluster.zip](https://github.com/project-chip/connectedhomeip/files/7785268/Media_cluster.zip)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_CC_3_1.yaml [hunk 1]
```diff
@@ -34,6 +34,15 @@ tests:
       response:
           value: 1
 
+    - label:
+          "Check current hue attribute value matched the value sent by the last
+          command"
+      disabled: true
+      command: "readAttribute"
+      attribute: "current hue"
```
Comment (jmeg-sfy) [inline src/app/tests/suites/certification/Test_TC_CC_3_1.yaml:42]: I would recommend to use CamelCase style only since it is the one use in the specs

**Review by jmeg-sfy — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_CC_3_1.yaml (hunk 1 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_CC_3_1.yaml:42]: Tried as per spec but YAML supporting different style
image

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml [hunk 2]
```diff
@@ -54,21 +54,12 @@ tests:
 
     #issue #11053 disabled steps below Global attributes missing from YAML framework
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:55]: Shouldn't this move down to the EventList bit?

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml [hunk 3]
```diff
@@ -54,21 +54,12 @@ tests:
...
     - label: "Read the global attribute: AttributeList"
-      disabled: true
       command: "readAttribute"
       attribute: "AttributeList"
       response:
           constraints:
               type: list
 
-    - label: "Read the global attribute: CommandList"
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:64]: Why is this being removed?

These comments seem to apply across the board.

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml (hunk 2 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:55]: I will update this

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml (hunk 3 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:64]: They removed it from spec

**Comment by bzbarsky-apple** [comment]
@manjunathambiger23 conflicts?

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml (hunk 3 above)
Comment (bzbarsky-apple) [reply src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:64]: It got renamed to ClientGeneratedCommandList/ServerGeneratedCommandList, no?

**Comment by manjunathambiger23** [comment]
> @manjunathambiger23 conflicts?

i will rebase it.

File: src/app/tests/suites/certification/Test_TC_BI_1_1.yaml (hunk 3 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_BI_1_1.yaml:64]: i added both ClientGeneratedCommandList/ServerGeneratedCommandList to scripts

**Comment by bzbarsky-apple** [comment]
@manjunathambiger23 failures?

**Pull request closed without merging**